x-pack/plugins/observability_solution/dataset_quality/public/hooks/use_telemetry.tsx

```diff
@@ -225,6 +231,16 @@ export const useDatasetDetailsTelemetry = () => {
     [ebtProps, telemetryClient]
   );
 
+  const trackDatasetDetailsBreakdownFieldChanged = useCallback(() => {
+    const datasetDetailsTrackingState = telemetryClient.getDatasetDetailsTrackingState();
+    if (datasetDetailsTrackingState === 'opened' && ebtProps) {
```

Comment:
```suggestion
    if (
        (datasetDetailsTrackingState === 'opened' || datasetDetailsTrackingState === 'navigated') &&
        ebtProps
      )
```
Because user can click <kbd>Cmd</kbd>+click a link and stay on the flyout and change breakdown field again. Currently it doesn't send the event this case.